lib/rubocop/formatter/formatter_set.rb

```diff
@@ -73,7 +76,7 @@ def add_formatter(formatter_type, output_path = nil)
           output = $stdout
         end
 
-        self << formatter_class.new(output)
+        self << formatter_class.new(output, @options)
```

Comment:
That is why the constructor for `BaseFormatter` is now declared as `def initialize(output, options = {})`.